bigtable-dataflow-parent/bigtable-beam-import/src/test/java/com/google/cloud/bigtable/beam/sequencefiles/EndToEndIT.java

```diff
@@ -17,25 +17,21 @@
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
+import com.google.cloud.bigtable.beam.TestHelper;
```

Comment:
From the logs for this test, I see:
```
2020-09-25 13:20:15 INFO  DataflowPipelineOptions$StagingLocationFactory:186 - No stagingLocation provided, falling back to gcpTempLocation
```

is this creating a temp bucket (and cleaning it up after)? Should we create another one?

Reply:
Additionally from the logs, `PackageUtil` creates a lot of logging statments. Can we turn down the logging level to reduce some of the noise?

Reply:
Finally, in the cleanup it looks like it's trying to delete a folder that already does not exist.

Reply:
> From the logs for this test, I see:
> 
> ```
> 2020-09-25 13:20:15 INFO  DataflowPipelineOptions$StagingLocationFactory:186 - No stagingLocation provided, falling back to gcpTempLocation
> ```
> 
> is this creating a temp bucket (and cleaning it up after)? Should we create a bucket explicitly instead?

No, these tests never create a bucket. If 'stagingLocation' path is not provided, a subfolder will be used in `tempLocation`

Reply:
> Additionally from the logs, `PackageUtil` creates a lot of logging statements. Can we turn down the logging level to reduce some of the noise?

As I can see tests are running with LoggingLevel=info, which IHMO produces reasonable amount of messages. `PackageUtil` is external class, therefore we can't update it to lessen the level of noising messages, we can only shut it up.
The messages from `PackageUtil` are not useless and they might become valueable for those who degug this test.
So, I would allow `PackageUtil` to talk.

Reply:
> 
> 
> Finally, in the cleanup it looks like it's trying to delete a folder that already does not exist.

I fixed it. Not sure if you like the fix.